frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx

```diff
@@ -35,15 +35,15 @@ class UserSuccessModal extends React.Component {
         {temporaryPassword ? (
           <PasswordSuccess user={user} temporaryPassword={temporaryPassword} />
         ) : (
-          <EmailSuccess isEeSsoConfigured={isSsoConfigured} user={user} />
```

Comment:
Ahhh. Nice catch. I must have missed that from the last time I refactored this file.